src/routes/event.tsx

```diff
@@ -73,7 +69,6 @@ const Event = ({ eventKey }: Props) => {
         </Heading>
         {newestIncompleteMatch && (
           <MatchDetailsCard
-            key={newestIncompleteMatch.key}
```

Comment:
I am not sure why you removed this attribute. Having a key attribute helps Preact with diffing, so that it knows the difference between when items were changed, added, reordered, or removed. I think there was some weird edge case where when this page rerendered (due to the newestIncompleteMatch switching), it might have reset the input on the event matches list, or something like that.

Reply:
I removed it because the `MatchDetailsCard` component doesn't have `key` as a property. I don't know what it's passing into.